AR and IE fixes, HQM and FTBU additions

Redquaker opened the pull request:
**Changelog:**

-Fixed the Advanced Rocketry guidance computer slot not appearing when accessing the GUI menu of an assembled rocket.

-Tweaked maximum transfer rates on Immersive Engineering cables such that they don't burn when using the intended number of power supply connectors. This allows you to use 4 LV and MV connectors and 2 HV connectors (full 8192 RF/t) for supplying power on a single circuit while being stable. Using more than these amounts still causes wires to burn. Previously, you had to limit LV and MV circuits to 3 connectors and HV circuits to 1 connector supplying power.

-Added item requirements to several HQM quests that were previously unable to be completed.
-Added a small quest reward for the "Oil Power" quest.
-Replaced the "Ice Cellar" quest reward with the correct type of ice for use in cellars.
-Edited some typos, inconsistencies, and outdated information found in HQM quest descriptions.

-Added an updated ore generation table to the in-game FTBU guide with an external link.

Redquaker committed:
Add files via upload

Ihysoal commented:
The AR jar was recompiled by me from source with two lines of code changed while retaining the same  version (1.4.0 build 20).

Tests yielded no problems with the recompiled server AR and the default client AR 1.4.0-20, but the reverse causes a disconnect when accessing the rocket (or just the guidance computer inside?). This is of course all moot if both sides have the new jar.

If it's a problem because the jars are retrieved from curseforge or something, we'll have to think of something else, like a PR to the AR repo, but I heard that wasn't a likely thing to happen.

wormzjl commented:
I don't think modifying mod sources at this stage is a good idea, you can take a look at the tweaker mod to see if you can asm stuffs in.
https://github.com/wormzjl/PFAA-Tweaks

Redquaker committed:
FTBU Guide Edits

Edited typos and outdated information, especially regarding the IHL electrolysis bath and the use of IHL cables. I added in a few notes to certain sections to give the reader a bit more information than before.

Commit:
Fixed typos in guide file names

Redquaker committed:
AR decompiled version

Redquaker committed:
I'm dumb and uploaded a .rar

Redquaker committed:
HQM Change

Edited another typo that was missed from before
